spec/openapi.yaml

```diff
@@ -1138,6 +1157,40 @@ components:
                 description: Path to package
                 example: "jo/packages"
 
+    Transfer:
+      type: object
+      properties:
+        type:
+          type: string
+          enum: [ NO_TRANSFER, FROM_WEB, FROM_CONNECT, FROM_PRINTER, FROM_SLICER, FROM_CLIENT, TO_CONNECT, TO_CLIENT ]
+          example: FROM_WEB
+        path:
+          type: string
+          example: "/PrusaLink gcodes/Spice_Harvester_0.3mm_PLA_MK3S_12m.gcode"
+        url:
+          type: string
+          example: "https://files.printables.com/media/prints/42/gcodes/42_b42-242-442-8142c-424242/spice_harvester_0.3mm_pla_mk3s_12m.gcode"
+        size:
+          type: string
+          example: 42424242
+        progress:
+          type: number
+          example: 42.2
```

Comment:
Why is progress in transfer in percents and progress in job info a fraction in range 0-1? Can we be consistent at least in our own API?

Reply:
I've fixed this in api/v1/job